packages/node/src/integrations/tracing/vercelai/instrumentation.ts

```diff
@@ -71,16 +72,27 @@ export class SentryVercelAiInstrumentation extends InstrumentationBase {
         const existingExperimentalTelemetry = args[0].experimental_telemetry || {};
         const isEnabled = existingExperimentalTelemetry.isEnabled;
 
-        // if `isEnabled` is not explicitly set to `true` or `false`, enable telemetry
-        // but disable capturing inputs and outputs by default
-        if (isEnabled === undefined) {
-          args[0].experimental_telemetry = {
-            isEnabled: true,
-            recordInputs: false,
-            recordOutputs: false,
-            ...existingExperimentalTelemetry,
-          };
-        }
+        const client = getCurrentScope().getClient();
+        const shouldRecordImportAndExports = client?.getIntegrationByName(INTEGRATION_NAME)
```

Comment:
l: shouldn't this rather be named

```suggestion
        const shouldRecordInputsAndOutputs = client?.getIntegrationByName(INTEGRATION_NAME)
```

?